storage/postgres/tree_storage.go

```diff
@@ -43,13 +44,16 @@ const (
 			SELECT n.subtree_id, max(n.subtree_revision) AS max_revision
 			FROM subtree n
 			WHERE n.subtree_id IN (` + placeholderSQL + `) AND
-			n.tree_id = ? AND n.subtree_revision <= ?
+			n.tree_id = <param> AND n.subtree_revision <= <param>
 			GROUP BY n.subtree_id
 		) AS x
 		INNER JOIN subtree
 		ON subtree.subtree_id = x.subtree_id
 		AND subtree.subtree_revision = x.max_revision
-		AND subtree.tree_id = ?`
+		AND subtree.tree_id = <param>`
+	insertTreeHeadSQL = `INSERT INTO tree_head(tree_id,tree_head_timestamp,tree_size,root_hash,tree_revision,root_signature)
+                 VALUES($1,$2,$3,$4,$5,$6)`
+	//selectTreeRevisionAtSizeOrLargerSQL = "SELECT tree_revision,tree_size FROM tree_head WHERE tree_id=$1 AND TreeSize>=$2 ORDER BY tree_revision LIMIT 1"
```

Comment:
Why is this commented out? Should it just be deleted?

Reply:
These were in here from the MySQL code I adapted this from.  I am unsure if they are for unimplemented code or no longer need to be around.  I didn't want to remove them just yet until I knew more about why they were written